### ISSUE TITLE
ERROR: Failed building wheel for qdldl

### ISSUE THREAD

**Issue by Hans-han** [cantaro86/Financial-Models-Numerical-Methods#10]
like the title suggested, I had issue running option 3
`python3.11.4 -m venv --prompt FMNM python-venv
source python-venv/bin/activate
python3 -m pip install --upgrade pip
pip install --requirement requirements.txt
pip install -e .`

my python version is 3.11.4 
I cloned the repo to local
the environment is MacOS Ventura M1 MacBook Air

I have a very limited debugging experience. 
So I googled the error and found stack overflow had similar question and I attempted the solution but was not able to solve the issue. 

I appreciate any help in advance

**Comment by cantaro86** [comment]
Thanks for reporting the issue.  
Unfortunately, I cannot reproduce your error. On my system, it works fine.
Using python 3.11.4 on a Mac M2, when I run `pip install --requirement requirements.txt` I get:  

```
Building wheel for qdldl (pyproject.toml) ... done
Created wheel for qdldl
```

I can recommend removing the folder `python-venv` and
- check that you have the right version:  `python --version`
- create again the virtual environment:
```
python3 -m venv --prompt FMNM python-venv 
source python-venv/bin/activate 
python3 -m pip install --upgrade pip
```
- and then install the requirements using:
`python3 -m pip install --no-cache-dir --requirement requirements.txt`
Maybe the problem is the cache

Please let me know if you still get the same error. 
If it still doesn't work, maybe conda is the way to go.

**Comment by cantaro86** [comment]
If that doesn't work, try to replace `requirements.txt` with `list_of_packages.txt`.

**Comment by Hans-han** [comment]
Thank you for the help ! Let me try and see if it runs

**Comment by Hans-han** [comment]
issue solved by `pip install cmake` under python 3.11.4
